### PR TITLE
remove network policy default value

### PR DESCRIPTION
**Pull request by OsamaKhan220**

### CODE REVIEW

**Comment by stakater-user** [comment]
@OsamaKhan220 Validation Successful

**Review by callum-stakater — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -542,6 +542,8 @@ To disable liveness or readiness probe, set value of `enabled:` to `false`.
 
 All notable changes to this project will be documented here
 
+## v1.2.0
```
Comment (hanzala1234) [inline README.md:545]: I think version will be 1.1.10

File: application/values.yaml [hunk 2]
```diff
@@ -711,39 +711,32 @@ externalSecret:
...
-  # Egress rules
-  egress:
-    - to:
-      - ipBlock:
-          cidr: 10.0.0.0/24
-      ports:
-      - protocol: TCP
-        port: 5978
+  # additionalLabels:
+  #   key: value
```
Comment (hanzala1234) [inline application/values.yaml:715]: only comment out values

**Comment by rasheedamir** [comment]
also @OsamaKhan220 if other PR gets merged before this one then don't forget to update the version in changelog

**Review by hanzala1234 — APPROVED** [review]

**Pull request merged**